pyscriptjs/src/utils.ts

```diff
@@ -108,6 +106,12 @@ function globalExport(name: string, obj: any) {
     _global[name] = obj;
 }
 
-
-
-export { addClasses, removeClasses, getLastPath, ltrim, htmlDecode, guidGenerator, showError, handleFetchError, readTextFromPath, inJest, globalExport, };
+export function getAttribute(el:Element, attr:string ):string | null {
```

Comment:
I guess a linter could fix issues with spacing etc. such as `(el:Element, attr:string )` --> `(el:Element, attr:string)` but it's minor.

But this kind of stuff might be present in other places.